src/content/changes/highlights/2022-06-17.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2022-06-17T10:00:00
+title: Highlights for the week ending June 17, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+## User interfaces
+
```

Comment:
```suggestion

With `kubectl gs` release [v2.14.0](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.14.0), the service priority cluster label can be set when using the `template cluster` command with the `--service-priority` flag.

You can also set the service priority label during cluster creation via the web UI, and inspect cluster labels in their raw format from the cluster details page.

```
@anvddriesch @gusevda please edit as you see fit 🙂

Reply:
LGTM